Sources/PointFree/Episode/Show.swift

```diff
@@ -45,29 +48,86 @@ let episodeResponse =
     }
 )
 
-let useCreditResponse =
-  fetchEpisodeForParam
+private func episodePageData(
```

Comment:
this function produces the data necessary to render either the old or new episode page. We already had `EpisodePageData`, which I duplicated to `NewEpisodePageData` that holds data specific to the new page, and then I map that data to an `Either<EpisodePageData, NewEpisodePageData>`. Once we launch we will be able to delete and clean up this stuff.